packages/automated_actions/pyproject.toml

```
@@ -9,14 +9,21 @@ requires-python = "~= 3.12.0"
 dependencies = [
     "celery[sqs]==5.4.0",
     "fastapi==0.115.12",
+    "gql==3.5.0",
     "httpx==0.28.1",
+    "hvac>=2.3.0",
```

Please pin the version
```suggestion
    "hvac==2.3.0",
```

---

https://github.com/app-sre/automated-actions/pull/51